### ISSUE TITLE
References to redcap_log_event need revision for REDCap 9.6.0

### ISSUE THREAD

**Issue by pbchase** [ctsit/auto_populate_fields#35]
REDCap 9.6.0 introduces a feature that shards future inserts to redcap_log_event by project_id. This will require a modification to this module where it reads from redcap_log_event at https://github.com/ctsit/auto_populate_fields/blob/46df577507601e7a5fbcd6bc7bd86e06fddcec76/ExternalModule.php#L113 

The advice from Vanderbilt is to modify code like this:

```
// Existing code
$sql = "select * from redcap_log_event where project_id = $project_id";
 
// Modified code
$log_event_table = method_exists('\REDCap', 'getLogEventTable') ? \REDCap::getLogEventTable($project_id) : "redcap_log_event";
$sql = "select * from $log_event_table where project_id = $project_id";
```

**Comment by pbchase** [comment]
For background, see https://community.projectredcap.org/articles/74511/modulehookplugin-developers-changes-coming-in-redc.html

**Comment by ChemiKyle** [comment]
Interesting, time to `grep -r modules -e 'redcap_log_event'` on a few servers.

**Comment by pbchase** [comment]
addressed by PR #36
